### ISSUE TITLE
Shouldn't getReact search only among descendents of the resq node passed by the previous command?

### ISSUE THREAD

**Issue by matfer** [abhinaba-ghosh/cypress-react-selector#137]
Reading the docs I understand that getReact searches only among the descendants of the resq node passed by the previous command. So for example `cy.getReact('A').getReact('B')` should search for B only among A's descendants. I got this right?

If yes, I added this test to calculator.spec.js, which is failing
```
  it('getReact should search only among descendants of the resq node passed by the previous command', () => {
    cy.getReact('t')
      .nthNode(2)
      .getReact('t')
      .nthNode(9)
      .getProps('name')
      .should('eq', '5');
  });
```
It should get the element at index 2, then among its descendants get the element at index 9 (the button '5'). Instead it gets the button '9', as if the second getReact searched for elements starting from the root component.

**Comment by abhinaba-ghosh** [comment]
Hi @matfer , you are correct. This is always the desired outcome in any test automation scenario. Currently, We are blocked due to this: https://github.com/abhinaba-ghosh/cypress-react-selector/issues/41

Happy to get help.

**Comment by abhinaba-ghosh** [comment]
Hi, this is now fixed with version 2.3.13 ( #251 )

Example - https://github.com/abhinaba-ghosh/cypress-react-chain-queries

Feel free to re-open the ticket if you find any issues.
